diesel/src/type_impls/primitives.rs

```diff
@@ -55,7 +55,10 @@ mod foreign_impls {
     struct U16Proxy(u16);
 
     #[derive(FromSqlRow)]
-    #[cfg_attr(any(feature = "mysql_backend", feature = "postgres"), derive(AsExpression))]
+    #[cfg_attr(
+        any(feature = "mysql_backend", feature = "postgres"),
```

Comment:
To check: Should `postgres` here be `postgres_backend`?

Reply:
That should be `postgres_backend`, right.
```suggestion
        any(feature = "mysql_backend", feature = "postgres_backend"),
```

Generally speaking anything postgres specific that does not rely on `libpq` should be behind the `postgres_backend`, so anything that is not related to `PgConnection`.